tests/sentry/utils/locking/backends/test_redis.py

```diff
@@ -29,7 +29,7 @@ def test_success(self):
         assert duration - 2 < float(client.ttl(full_key)) <= duration
 
         self.backend.release(key)
-        assert client.exists(full_key) is False
+        assert not client.exists(full_key)
```

Comment:
Redis itself now accepts many keys to this, and returns the number that exist. So instead of true/false, for one key this will return 1/0. Behaviour is the same for truthiness checking.

Reply:
Wouldn't this cause a type-safety issue in Py3? Should we just do `assert client.exists(full_key) == 0` instead?

Reply:
I don't think so? If you're worried about type hinting etc I think we should leave that as a separate project